pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -48,9 +48,11 @@ type caOptions struct {
 	ExternalCAType   ra.CaExternalType
 	ExternalCASigner string
 	// domain to use in SPIFFE identity URLs
-	TrustDomain      string
-	Namespace        string
-	Authenticators   []security.Authenticator
+	TrustDomain    string
+	Namespace      string
+	Authenticators []security.Authenticator
+	// CertSignerDomain is based on CERT_SIGNER_DOMAIN, combined with the user-supplied name
+        // to form the K8S signer for RA.
```

Comment:
I completely agree - but someone must do it in a separate PR and we need to find who's using it. 

It's a VERY dangerous option - CertManager will be messed up since there is no check or namespace isolation.
We need to trace who added the option and if there are users - as usual far harder to remove than add.